### PR TITLE
Fix UI scale being applied unevenly, causing elements to move around relative to each other

### PR DESCRIPTION
**Pull request by peppy**
Before:

https://user-images.githubusercontent.com/191335/163363029-67f001cd-36f9-443c-9708-5c520f0e2dc7.mp4

After:

https://user-images.githubusercontent.com/191335/163362880-1314c137-28e9-4102-bbd4-72fc0d7cc38f.mp4

### CODE REVIEW

File: osu.Game/Graphics/Containers/ScalingContainer.cs [hunk 1]
```diff
@@ -101,8 +105,15 @@ private void load(OsuConfigManager osuConfig)
 
             private void scaleChanged(ValueChangedEvent<float> args)
             {
-                this.ScaleTo(new Vector2(args.NewValue), 500, Easing.Out);
-                this.ResizeTo(new Vector2(1 / args.NewValue), 500, Easing.Out);
+                this.TransformBindableTo(currentScale, args.NewValue, duration, Easing.OutQuart);
```
Comment (smoogipoo) [inline osu.Game/Graphics/Containers/ScalingContainer.cs:108]: Not a big deal, but you could use `this.TransformTo(nameof(currentScale))`.

**Review by smoogipoo — DISMISSED** [review]

File: osu.Game/Graphics/Containers/ScalingContainer.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/Graphics/Containers/ScalingContainer.cs:108]: is that more preferred for some reason?

Comment (peppy) [reply osu.Game/Graphics/Containers/ScalingContainer.cs:108]: oh, to not make it a bindable? i didn't even consider that.

Comment (smoogipoo) [reply osu.Game/Graphics/Containers/ScalingContainer.cs:108]: Doesn't make much of a difference at the end of the day, just avoids a bindable alloc.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**